scripts/ObstacleDetector.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python
```

Comment:
Might need a different, more descriptive name, something implying 2D information (since we're making a 3D version of this)